invenio_sequencegenerator/templates/invenio_sequencegenerator/custom_list.html

```diff
@@ -0,0 +1,36 @@
+{#
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.}
+#}
+{%- extends 'admin/model/list.html' %}
+{%- block list_row_actions %}
+  {{ super() }}
+  <form class="icon" method="POST" action="{{ url_for('.reset_view') }}">
+    <input name="start" value="0" type="number" size="1">
+    <input name="rowid" value="{{ get_pk_value(row) }}" type="hidden">
+    <button onclick="return confirm('Are you sure you want to reset selected counter?');" title="Reset">
```

Comment:
please make sure that all strings are marked for translations with `{{ _('...') }}`